services/workflows-service/src/common/abstract-logger/abstract-logger.ts

```diff
@@ -7,6 +7,6 @@ export abstract class IAppLogger {
   abstract info(message: string, payload: LogPayload): void;
   abstract warn(message: string, payload: LogPayload): void;
   abstract debug(message: string, payload: LogPayload): void;
-  abstract error(message: string, payload: LogPayload): void;
+  abstract error(message: Error, payload: LogPayload): void;
```

Comment:
I'd expect the signature to be the same as `window.console.error` or at least what we use in the workflows service.
![image](https://github.com/ballerine-io/ballerine/assets/61207713/b0ac71f2-9473-4ad5-be9b-c93b863da0e4)

Reply:
changed to error | string